### ISSUE TITLE
Question: Is this plugin Strapi Backen UI Only?

### ISSUE THREAD

**Issue by MokDevelopment** [notum-cz/strapi-plugin-record-locking#16]
Is this plugin Strapi Backen UI Only?
Could not find that bit on the info page.

**Comment by Fikoun** [comment]
Yes, as is for now, the plugin only protects from cross editing in **Strapi Admin**. 
This is a great point though and there will be good thought given to how it should be implemented for API.

**Comment by MokDevelopment** [comment]
Yeah would be nice to have an endpoint that delivers via type/id true or false. In my case the admin is really just used for the configuration. So this feature is really missing :)

**Comment by omikulcik** [comment]
Hi @MokDevelopment, I am now maintaining this plugin. I agree that this would be a valuable feature. We will consider adding it to our roadmap. It would be also appreciated if someone is willing to contribute by opening a PR with this feature.

**Comment by JersonDiaz025** [comment]
> Hola[@MokDevelopment](https://github.com/MokDevelopment)Actualmente estoy a cargo del mantenimiento de este plugin. Estoy de acuerdo en que sería una función valiosa. Consideraremos añadirla a nuestra hoja de ruta. También agradeceríamos que alguien colaborara abriendo una solicitud de registro (PR) con esta función.

Hola, esto no me funciona para strapi v5
